src/i18n/es.js

```diff
@@ -59,6 +59,8 @@ export default {
   change_plan: {
     all_of_plan: `Todo lo de {plan} más:`,
     ask_demo: 'Solicita una demo',
+    banner_exclusive_features_description: 'Amplifique su programa de correo electrónico con el apoyo adicional de nuestro equipo.',
```

Comment:
```suggestion
    banner_exclusive_features_description: 'Amplifica tu programa de envío de mails con el apoyo adicional de nuestro Equipo.',
```
@mrunge86  what do you think?

Reply:
fixed